thoth/package_release_job/__init__.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# thoth-package-release-job
+# Copyright(C) 2018, 2019, 2020 Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth package-release-job."""
+
+from thoth.common import __version__ as __common__version__
+from thoth.common import init_logging
+from thoth.storages import __version__ as __storages__version__
+from thoth.python import __version__ as __python__version__
+
+__version__ = "0.6.0"
+__service_version__ = f"{__version__}+\
```

Comment:
You can put these values directly to `app.py` no need to create this directory structure.

Reply:
Don't we want to run https://github.com/thoth-station/kebechet/tree/master/kebechet/managers/version#kebechet-version-manager here?

Reply:
Yes, we want. It respects openshift s2i conventions and checks `app.py` for version management.